### ISSUE TITLE
Dropping files into a dropfolder creates versions of the file

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#7643]
### Steps to reproduce
1. Create a folder `Homework`
2. Share `Homework` via link (permission: "Upload only")
3. open the link, drop `Sheet.txt`
4. edit `Sheet.txt`, save, drop `Sheet.txt` again
5. `Homework` contains `Sheet.txt` with 1 version

![Screenshot 000563@2x](https://user-images.githubusercontent.com/26610733/190340240-09547c15-6471-4f49-b7fc-cc4d55bf6e73.png)

### Expected behaviour
- `Homework` contains `Sheet.txt` and `Sheet (1).txt`

![Screenshot 000564@2x](https://user-images.githubusercontent.com/26610733/190340304-9d88b1d1-f7b0-4cfa-8765-75cdd7e43cba.png)
 

### Actual behaviour
- `Homework` contains `Sheet.txt` with 2 versions

**Comment by kulmann** [comment]
@micbar my understanding is that ocis needs to create a second file (like described above in the expected behaviour) when the upload comes via an `upload only` public share. Can you confirm that?

**Comment by phil-davis** [comment]
> my understanding is that ocis needs to create a second file

yes, that is what is supposed to happen - similar to what oC10 does.

Also see recent discussion in https://github.com/owncloud/ocis/issues/5074 - the way that unique filenames are generated by the oCIS/reva backend is being discussed.

**Comment by LukasHirt** [comment]
@kobergj this sounds like oCIS issue then? Or is there anything extra that Web should do when uploading files in the upload-only view? Otherwise I'd suggest moving this into oCIS repo.
